_rules/image-non-empty-accessible-name-23a2a8.md

```diff
@@ -146,6 +151,33 @@ This `img` element inside a `div` positioned off screen has an [implicit role][]
 </div>
 ```
 
+#### Passed Example 9
+
+This `img` element displays a CSS `background-image` and has an [accessible name][] because of the `alt` attribute.
+
+```html
+<img alt="W3C logo" style="width:72px; height:48px; background-image: url(/test-assets/shared/w3c-logo.png)" />
+```
+
+#### Passed Example 10
+
+This `img` element displays a `srcset` and has an [accessible name][] because of the `alt` attribute.
+
+```html
+<img alt="W3C logo" srcset="/test-assets/shared/w3c-logo.png" />
+```
+
+#### Passed Example 11
+
+This `picture` element displays a `srcset` and has an [accessible name][] because of the `alt` attribute.
+
+```html
+<picture>
+    <source srcset="/test-assets/shared/w3c-logo.png" />
+    <img alt="W3C logo" width="72" height="48" />
+</picture>
+```
```

Comment:
Technically, the rule is applicable to the `img` element, not to the `picture` one.
It doesn't really change anything as the `picture` element is just here to give the `source` to the `img`, but description should be about the `img` element.

Reply:
I've changed both `picture` examples to say "This `img` element displays"